src/components/Products-section/ProductsSection.jsx

```diff
@@ -1,5 +1,5 @@
 import "./productsSection.css";
-import { ProductCard } from "../Product-card/ProductCard";
+import { ProductCardListing } from "../Product-card-listing/ProductCardListing";
 import { useFilterData } from "../../contexts/filterData-context";
 import { PopUpFiltersContainer } from "../PopUp-filters/PopUpFilters";
```

Comment:
You can make use of barrel imports for components

Reply:
okk didn't thought of doing it, thanks for pointing that out!!